src/Microsoft.ML.Data/DataView/DataViewExtensions.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Microsoft.ML.Data
+{
+    public static class DataViewExtensions
+    {
+        /// <summary>
+        /// Get a row cursor. The active column indices are those for which needCol(col) returns true.
+        /// The schema of the returned cursor will be the same as the schema of the IDataView, but getting
+        /// a getter for an inactive columns will throw. The <paramref name="namesOfcolsNeeded"/> predicate must be
+        /// non-null.If set to <langword>null</langword> no column is active.
```

Comment:
>langword [](start = 32, length = 8)

Is `<langword>` a thing? In VS at least while I can use its facilities to autocomplete to `<see langword` (which is what I've seen used elsewhere), I do not see that `<langword>` just itself is a thing. Though, you have more experience with the doc system than I do perhaps, so maybe I just somehow missed it? Do you have a reference?

Reply:
thanks for the catch! should switch to <see langword 

---
In reply to: [245081549](https://github.com/dotnet/machinelearning/pull/1975#discussion_r245081549) [](ancestors = 245081549)